src/components/pages/Messages/Messages.js

```diff
@@ -21,12 +23,24 @@ class Messages extends React.Component {
       });
   }
 
+  deleteSingleMessage = (messageId) => {
+    messageRequests.deleteMessage(messageId)
```

Comment:
when you do a firebase delete it doesn't return the remaining messages.  don't you need to recall smashRequests.getAllMessagesWithUserInfo() here?  

seems like that would be duplication of lines 14-23...maybe break that stuff out into another function and call that function here and in the componentDidMount

Reply:
DOH! That was stupid. It was there before I did a last minute 'tidy-up'. I recognized that I'll be using the same code. Was going to do a new card for refactor. Should I go ahead and refactor that now?

Reply:
looking good!